src/content/ja/updates/2017/04/headless-chrome.md

```diff
@@ -0,0 +1,312 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ヘッドレス Chrome とはなにか、そしてその使い方を紹介します。
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
+
+# ヘッドレス Chrome ことはじめ {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+</style>
+
+### TL;DR {: #tldr .hide-from-toc}
+
+[Headless Chrome](https://chromium.googlesource.com/chromium/src/+/master/headless/README.md) は Chrome をヘッドレス環境で実行する手段です。Chrome をクローム（ブラウザーのUIのこと）なしに実行します！ヘッドレス Chrome によって、Chromium とそのエンジン Blink が提供する**モダンなウェブプラットフォームの機能すべて**がコマンドラインにもたらされるのです。
+
```

Comment:
doing so in Japanese isn't great as line breaks create whitespaces and that's awkward to the Japanese.